app/src/main/kotlin/com/wire/android/ui/common/WireDialog.kt

```diff
@@ -176,10 +180,17 @@ private fun WireDialogContent(
                 modifier = Modifier.padding(bottom = MaterialTheme.wireDimensions.dialogTextsSpacing)
             )
             text?.let {
-                Text(
+                ClickableText(
                     text = text,
                     style = MaterialTheme.wireTypography.body01,
-                    modifier = Modifier.padding(bottom = MaterialTheme.wireDimensions.dialogTextsSpacing)
+                    modifier = Modifier.padding(bottom = MaterialTheme.wireDimensions.dialogTextsSpacing),
+                    onClick = { offset ->
+                        text.getStringAnnotations(
+                            tag = MarkdownConsts.TAG_URL,
```

Comment:
Not being the focus on this PR but can `MarkdownConsts` be renamed correctly to `MarkdownConstants`?